### DISCUSSION TITLE
Not Grabbing all Keyboard Input

### DISCUSSION THREAD

**Discussion by KenHolloway** [FreeRDP/FreeRDP#10419] [Q&A]
Some time in the past couple of months, something changed. FreeRDP is no longer grabbing all keyboard input. So, for instance, Alt-F4 is causing the session to drop, instead of passing Alt-F4 to the remote computer.

Any ideas how to find out what changed, and more importantly, how to fix it?

I am running "FreeRDP version 2.11.8-dev (2.11.8)" on "openSUSE Tumbleweed", connecting to numerous different versions of Windows.

**Comment by akallabeth** [comment]
@KenHolloway no big changes on oldstable anymore.
you could run a `git bisect` between the version you last had working and now.
also your environment might have to do with that (`X11`? `XWayland`?...)

**Reply by akallabeth** [reply]
also, can not reproduce with `2.10.0` from debian repos (diff to your version does not contain any functional changes)
did run in gnome/wayland with `xfreerdp /v:host /u:user /p:pwd`
